pkg/placement/controller.go

```diff
@@ -325,6 +335,20 @@ func (c *Controller) enqueueObject(obj interface{}, skipCheckIsDeleted bool) {
 	c.workqueue.Add(key)
 }
 
+func (c *Controller) enqueuePlacementDecision(name string) {
+	c.workqueue.Add(util.Key{
```

Comment:
FYI, `workqueue.AddAfter` is a thing. This controller could [Nagle](https://en.wikipedia.org/wiki/Nagle%27s_algorithm).